uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -72,6 +72,30 @@ define([
       }
 
       /**
+       * Load file extensions which are supported for upload
+       *
+       * @return {Promise}
+       */
+      function loadSupportedFileTypes () {
+        return OptionGroup.valuesOf('safe_file_extension')
+          .then(function (extensions) {
+            var allowedMimeTypes = {};
+            extensions.map(function (ext) {
+              allowedMimeTypes[ext.label] = sharedSettings.fileUploader.mimeTypesMap[ext.label];
```

Comment:
That is why I have left a comment in the code saying "If other extensions are added in the OptionGroups, mime types need to be added here", so if we need to add more extensions we need to update this list here. I dont see any work around to this unfortunately.